inc/ticket.class.php

```diff
@@ -323,7 +323,7 @@ static function addHistoryOnAddGroup(CommonDBTM $item) {
       return $item;
    }
 
-   static function processAfterAddGroup(CommonDBTM $item) {
+   static function processAfterAddGroup(CommonDBTM &$item) {
```

Comment:
In PHP, objects are always passed by reference.
```suggestion
   static function processAfterAddGroup(CommonDBTM $item) {
```